app/MMB/src/scenes/tripServices/insurance/InsuranceOverviewScene.js

```diff
@@ -0,0 +1,135 @@
+// @flow strict
+
+import * as React from 'react';
+import { ScrollView, View } from 'react-native';
+import { LayoutSingleColumn, TextButton } from '@kiwicom/mobile-shared';
+import { graphql, PrivateApiRenderer } from '@kiwicom/mobile-relay';
+import { Translation } from '@kiwicom/mobile-localization';
+import {
+  TitledMenuGroup,
+  MenuItem,
+  type RouteNamesType,
+  type NavigationType,
+} from '@kiwicom/mobile-navigation';
+
+import DestinationImage from './DestinationImage';
+import OrderSummary from './OrderSummary';
+import TripInfo from '../../../components/header/TripInfo';
+import BookingDetailContext from '../../../context/BookingDetailContext';
+import type { InsuranceOverviewSceneQueryResponse } from './__generated__/InsuranceOverviewSceneQuery.graphql';
+
+type Props = {|
+  data: InsuranceOverviewSceneQueryResponse,
+  navigation: NavigationType,
+|};
+
+class InsuranceOverviewScene extends React.Component<Props> {
+  navigate = (routeName: RouteNamesType) => {
+    this.props.navigation.navigate({
+      routeName,
+      key: 'key-' + routeName,
+    });
+  };
+
+  goToTheInsuranceSelection = () => {
+    // TODO: PAX ID (?)
+    this.navigate('mmb.trip_services.insurance.selection');
+  };
+
+  goToTheInsurancePayment = () => {
+    this.navigate('mmb.trip_services.insurance.payment');
+  };
+
+  goToTheInsuranceRefund = () => {
+    this.navigate('mmb.trip_services.insurance.refund');
+  };
+
+  render = () => {
+    const { data } = this.props;
+    return (
+      <React.Fragment>
+        <ScrollView>
+          <LayoutSingleColumn>
+            <DestinationImage data={data.node} />
+            <TripInfo data={data.node} />
+            <TitledMenuGroup
+              title={<Translation id="mmb.trip_services.order.pax" />}
+            >
+              <MenuItem
+                title={
+                  <Translation passThrough="TODO: list of PAX (links to the insurance selection)" />
+                }
+                onPress={this.goToTheInsuranceSelection}
+              />
+              <MenuItem
+                title={
+                  <Translation passThrough="TODO: list of PAX (links to the insurance selection)" />
+                }
+                onPress={this.goToTheInsuranceSelection}
+              />
+            </TitledMenuGroup>
+            <View style={{ padding: 10 }}>
```

Comment:
Do you want your style to be inline here?

Reply:
You are right it should be done differently. However, this is all WIP so can we skip it here, please? 🙂